autotest/gdrivers/hdf5.py

```diff
@@ -502,6 +515,7 @@ def test( self ):
     hdf5_11,
     hdf5_12,
     hdf5_13
```

Comment:
Missing comma after hdf5_13

Reply:
I am flying blind on the testsuite. I wasn't able to get the test harness running. Is there documentation anywhere that explains what is needed to run the testsuite?